modules/power-monitoring-hardware-virtualization-support.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * power_monitoring/power-monitoring-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="power-monitoring-hardware-virtualization-support_{context}"]
+= {PM-kepler} hardware and virtualization support
+
+{PM-kepler} is the key component of {PM-shortname} that collects real-time power consumption data from a node through one of the following power estimation methods:
+
+Kernel Power Management Subsystem (preferred)::
+* `rapl-sysfs`: This requires access to the `/sys/class/powercap/intel-rapl` host file.
+* `rapl-msr`: This requires access to the `/dev/cpu/*/msr` host file.
+
```

Comment:
Is this consider a power estimation method?

Reply:
No, none of the methods under `kernel Power Management Subsystem` (reading values directly exposed by kernel) uses estimator.

Reply:
I'm asking because the text is implying this.  See:  
https://70711--ocpdocs-pr.netlify.app/openshift-enterprise/latest/power_monitoring/power-monitoring-overview#:~:text=Kepler%20is%20the%20key%20component%20of%20power%20monitoring%20that%20collects%20real%2Dtime%20power%20consumption%20data%20from%20a%20node%20through%20one%20of%20the%20following%20power%20estimation%20methods%3A